### PR TITLE
UI/control group db cred

### PR DESCRIPTION
**Pull request by hashishaw**
Fix a bug where control groups were not working properly on database credential paths. It now works as expected (approver on left, requester on right):

database-creds-cg

The language for "Generate credentials" has also been updated for static type roles only:
Screen Shot 2021-07-08 at 2 18 44 PM
Screen Shot 2021-07-08 at 2 18 47 PM

Screen Shot 2021-07-08 at 2 18 52 PM
Screen Shot 2021-07-08 at 2 18 59 PM

### CODE REVIEW

File: ui/app/components/generate-credentials-database.js [hunk 1]
```diff
@@ -8,82 +8,19 @@
...
-  @tracked errorTitle = 'Something went wrong';
-
-  constructor() {
-    super(...arguments);
-    this.fetchCredentials.perform();
+  get errorTitle() {
+    return this.args.model.errorTitle || 'Something went wrong';
   }
 
-  @task(function*() {
```
Comment (Monkeychip) [inline ui/app/components/generate-credentials-database.js:38]: much cleaner, thank you for moving this logic out of the component.

File: ui/app/templates/components/database-role-edit.hbs [hunk 2]
```diff
@@ -38,7 +38,7 @@
         {{on 'click' (fn this.generateCreds @model.id)}}
         data-test-database-role-generate-creds
       >
-        Generate credentials
+        {{if (eq @model.type "static") "Get credentials" "Generate credentials"}}
```
Comment (Monkeychip) [inline ui/app/templates/components/database-role-edit.hbs:41]: thank you for tackling this while you're in here

File: ui/app/templates/components/generate-credentials-database.hbs [hunk 3]
```diff
@@ -18,15 +18,15 @@
   </p.levelLeft>
```
Comment (Monkeychip) [inline ui/app/templates/components/generate-credentials-database.hbs:18]: glimmerizing! 💙

File: ui/app/templates/components/get-credentials-card.hbs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<div class="selectable-card is-rounded no-flex">
```
Comment (Monkeychip) [inline ui/app/templates/components/get-credentials-card.hbs:1]: well done. Should have been this in the first place, thank you.

**Review by Monkeychip — APPROVED** [review]
Thank you! And thank you for cleaning up old work while you're in there.

File: ui/app/templates/components/get-credentials-card.hbs (hunk 4 above)
Comment (hashishaw) [reply ui/app/templates/components/get-credentials-card.hbs:1]: One of those things you don't catch until you're actually using it 😅

**Pull request merged**